twisted/internet/udp.py

```diff
@@ -311,6 +311,18 @@ def write(self, datagram, addr=None):
 
 
     def writeSequence(self, seq, addr):
+        """
+        Write a datagram constructed from a L{list} of L{bytes}.
+
+        @param seq: The data that will make up the complete datagram to be
+            written.
+        @type seq: L{list} of L{bytes}
+
+        @type addr: L{tuple} containing L{str} as first element and L{int} as
+            second element, or L{None}
+        @param addr: A tuple of (I{stringified IPv4 or IPv6 address},
+            I{integer port number}); can be L{None} in connected mode.
+        """
```

Comment:
Yes, ticket updated per review process instructions (not sure if I should ack PR's in Trac or here or both)